test/integration_test.go

```diff
@@ -892,7 +892,6 @@ func TestAutoRetryFlagWithRecoverableError(t *testing.T) {
 }
 
 func TestAutoRetryEnvVarWithRecoverableError(t *testing.T) {
-	t.Parallel()
 	os.Setenv("TERRAGRUNT_AUTO_RETRY", "false")
```

Comment:
Perhaps a better fix is to not set this as an env var and instead just pass it in as a CLI arg?

Reply:
There is already a test testing the cli arg `TestAutoRetryFlagWithRecoverableError`. Just wanted to cover the env var case as well.

Reply:
Ah, OK, gotcha